src/test/java/teammates/test/cases/browsertests/TableSortTest.java

```diff
@@ -132,6 +132,27 @@ public void testTableSortingPoint() {
                 "N/S",
                 "N/A");
     }
+    
+    @Test
+    public void testTableSortingPointNumber() {
+        verifySortingOrder(By.id("button_sortPointNumber"),
+        
+                "-1.667",
+                "-1.51",
+                "-1",
+                "-0.5",
+                "-0.4",
+                "-0.1",
+                "0",
+                "0.2",
+                "0.333",
+                "0.45",
+                "0.9",
+                "1",
+                "1.1",
+                "1.333",
+                "1.45");
+    }
```

Comment:
I have checked for that. It will fail. :)
The test cases are selected for the purpose as if we use `parseInt` instead of `parseFloat`, both `1.333` and `1.45` will become `1` and there is no difference between them.